pkg/tcl/testworkflowstcl/testworkflowprocessor/container.go

```diff
@@ -421,6 +422,13 @@ func (c *container) EnableToolkit(ref string) Container {
 	})
 }
 
+func (c *container) RunAsRoot() Container {
```

Comment:
this might produce questions for OpenShift and other rootless confogurations

Reply:
Sure thing, but we already have this problem - current artifacts scraper also works as a root:
* [**Dockerfile**](https://github.com/kubeshop/testkube/blob/6daf1ab9cada7969a2f3cfe667d8706244089d97/contrib/executor/scraper/build/agent/Dockerfile) is using default Alpine user == root
* [**Template**](https://github.com/kubeshop/helm-charts/blob/develop/charts/testkube-api/job-scraper-template.yml.tmpl) doesn't have any `securityContext` overrides, so it keeps this way

The problem is, that you need to have permissions to read the artifacts, so you either need to be root user, or you need to use same user for all the steps, or you need to apply proper permissions in each step